api/db/apd.test.js

```diff
@@ -0,0 +1,37 @@
+const tap = require('tap');
+const sinon = require('sinon');
+
+const apd = require('./apd')();
+
+tap.test('state data model', async apdModelTests => {
```

Comment:
Copy/pasta artifact!  Test name refers to the state data model.  I do this a bunch.  😬